src/OAuthAuthenticator.cs

```diff
@@ -14,10 +14,25 @@ public class OAuthAuthenticator
     {
         private readonly string _apiKey;
         private readonly string _secret;
-        public OAuthAuthenticator(string apiKey, string secret)
+        private readonly string _versionParameter;
+
+        public OAuthAuthenticator(string apiKey, string secret, OAuthVersionParameter versionParameter = OAuthVersionParameter.OneZeroA)
         {
             _apiKey = apiKey;
             _secret = secret;
+            switch (versionParameter)
```

Comment:
See comment below -- let's put this in a method in the https://github.com/AlexGhiondea/OAuth.net/blob/master/src/Helpers/OAuthHelpers.cs file